src/Misc/layoutbin/en-US/strings.json

```diff
@@ -168,6 +169,10 @@
   "ProcessExitCode": "Exit code {0} returned from process: file name '{1}', arguments '{2}'.",
   "Prompt0": "Enter {0}",
   "Prompt0Default1": "Enter {0} (press enter for {1})",
+  "QueueConError": "Message Queue Connect Error: {0}. Retrying in {1} sec.",
```

Comment:
Since we're just showing first error (right?) maybe we should say Retring in x sec.  Instead, say ... will retry every x seconds untils reconnected.  Then when reconnected, will see message